src/qt/dash.cpp

```diff
@@ -740,6 +740,39 @@ int main(int argc, char *argv[])
         }
         GUIUtil::setFontScale(nScale);
     }
+    // Validate/set custom css directory
+    if (gArgs.IsArgSet("-custom-css-dir")) {
+        fs::path customDir = fs::path(gArgs.GetArg("-custom-css-dir", ""));
+        QString strCustomDir = QString::fromStdString(customDir.string());
+        std::vector<QString> vecRequiredFiles = GUIUtil::listStyleSheets();
+        QString strMissing, strFile;
+
+        if (!fs::is_directory(customDir)) {
+            QMessageBox::critical(0, QObject::tr(PACKAGE_NAME),
+                                  QObject::tr("Error: Invalid -custom-css-dir path.") + "\n\n" + strCustomDir);
```

Comment:
Why the double new line?

Reply:
To get a bit space before the list of missing files. Thanks for asking haven't tested that for a while and it now realised that i messed with it over the time, will force push soon. 
<img width="532" alt="Bildschirmfoto 2020-07-11 um 05 07 14" src="https://user-images.githubusercontent.com/35775977/87215212-85479c80-c334-11ea-967f-5a36f8041dd6.png">